fdbserver/LogSystemPeekCursor.actor.cpp

```diff
@@ -585,6 +585,43 @@ static void resetBestServerIfNotAvailable(
 	}
 }
 
+static bool canReturnEmptyVersionRange(
+    int bestServer,
+    int currentServer,
+    Version end,
+    const Optional<std::vector<uint16_t>>& knownLockedTLogIds = Optional<std::vector<uint16_t>>(),
```

Comment:
For readability, can this be renamed to `knownLockedTLogIndices` ?  I associate tLog IDs with unique UIDs.